petab/problem.py

```diff
@@ -347,66 +347,111 @@ def get_noise_distributions(self):
         return measurements.get_noise_distributions(
             measurement_df=self.measurement_df)
 
+    def _to_mask(v: List, free: bool = True, fixed: bool = True):
```

Comment:
What's `v`?

Reply:
any vector. maybe `arr` instead

Reply:
Not so much about the name, but document arguments.